compiler/modules/comm.diagnostic/printer.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/io;
+import ballerina/file;
+
+public type Error error<Diagnostic>;
+
+public type Printer object {
+    public function print(Error diagnostic);
+    public function flushPrinter() returns (file:Error|io:Error)?;
```

Comment:
`flushPrinter` does not pair well with `print`. I suggest something like `finish` or `close`.

Reply:
fixed